### ISSUE TITLE
Empty file after DocumentUpload.create() and dispute.addFileEvidence()

### ISSUE THREAD

**Issue by kkuuko** [braintree/braintree_node#127]
### General information

* SDK/Library version: 2.5.0
* Environment: Sandbox
* Language, language version, and OS: Node v8.4.0 running within docker container build from the official image for nodejs

### Issue description

First I upload a pdf file to Braintree via DocumentUpload.create() and attach the uploaded file to the dispute using dispute.addFileEvidence(). The file is uploaded and exists in the Braintree Control Panel. It contains the same number of pages as the original document being uploaded but without any content. I tried the same with a png image ending up with the same result.

DocumentUpload.create():
* Request
```
gateway.documentUpload.create({
    kind: 'evidence_document',
    file: fs.createReadStream(pdfFilePath)
}).then(function (response) {
```
* Response
```
{ documentUpload:
   DocumentUpload {
     id: '412574e6-577c-4543-879b-c6f1cb589b6a',
     contentType: 'application/pdf',
     kind: 'evidence_document',
     name: '85952ed0-128f-11e8-8eb8-314188d6586c.pdf',
     size: 19111,
     expiresAt: '2018-04-16' },
  success: true }
```

dispute.addFileEvidence():
* Request
```
gateway.dispute.addFileEvidence(
    dispute_id,
    documentUpload.id
).then(function (response) {
```
* Response
```
{ evidence:
   { id: 'vp6h4x9tjwbt6kxz',
     category: null,
     comment: null,
     createdAt: '2018-02-15T20:33:50Z',
     sentToProcessorAt: null,
     sequenceNumber: null,
     url: 'https://bt-sandbox-arbiter-evidence-files.s3.amazonaws.com/...' },
  success: true }
```

**Comment by sdcoffey** [comment]
Hey @kkuuko,

Thanks for submitting this bug report. This is a bug in the SDK. We're currently working on fixing it, and will let you know we've released a new version with the fix.

**Comment by skunkworks** [comment]
Hi @kkuuko,

This issue was fixed in [2.8.0](https://github.com/braintree/braintree_node/blob/master/CHANGELOG.md#280). Let us know if that resolves your issue!

**Comment by kkuuko** [comment]
Thanks, now its working.
